pom.xml

```diff
@@ -4,36 +4,87 @@
 
   <groupId>org.jlab.clas.timeline</groupId>
   <artifactId>clas12-timeline</artifactId>
-  <version>1.0-SNAPSHOT</version>
+  <version>2.99.0-SNAPSHOT</version>
   <name>clas12-timeline</name>
   <url>http://www.github.com/JeffersonLab/clas12-timeline</url>
 
-  <packaging>pom</packaging>
-
-  <modules>
-    <module>monitoring</module>
-    <module>detectors</module>
-  </modules>
+  <packaging>jar</packaging>
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <release>21</release>
   </properties>
 
+  <repositories>
+    <repository>
+      <id>clas12maven</id>
+      <url>https://clasweb.jlab.org/clas12maven</url>
+    </repository>
+  </repositories>
+
+  <pluginRepositories>
+    <!-- for latest `groovy-eclipse-*` plugins -->
+    <pluginRepository>
+      <id>groovy-plugins-release</id>
+      <url>https://groovy.jfrog.io/artifactory/plugins-release</url>
+    </pluginRepository>
```

Comment:
this is blocked on `ifarm`:
```
>>> curl https://groovy.jfrog.io/artifactory/plugins-release/
curl: (35) OpenSSL SSL_connect: Connection reset by peer in connection to groovy.jfrog.io:443
```

Reply:
we indeed need this repo, since [this is where they release](https://github.com/groovy/groovy-eclipse/wiki/Groovy-Eclipse-Maven-plugin) and this [seems to be the recommended way to build a Java+Groovy application with Maven](https://www.baeldung.com/groovy-java-applications).

Note that on `mvnrepository.com`, both `groovy-eclipse-compiler` and `groovy-eclipse-batch` are a major version behind and 4 years old.